project/plugins.sbt

```diff
@@ -7,7 +7,7 @@ addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.3.4")
 addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.6.0")
 addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-site" % "1.3.3")
+addSbtPlugin("com.typesafe.sbt" % "sbt-site" % "1.4.1")
```

Comment:
@SethTisue Either something is wrong with recent sbt-site, or the settings needs more changes, I can no longer build the documentation, so I need to downgrade back to sbt-site 1.3.3.

Reply:
fine w/ me :-)

Reply:
https://github.com/sbt/sbt-jacoco/pull/177 I think is the fix.